README.md

```diff
@@ -419,6 +419,14 @@ The add the following to your `.gitconfig` (usually found at `~/.gitconfig`).
   textconv = sh -c 'sphobjinv convert plain "$0" -'
 ```
 
+### Dependabot - upgrade notebook testing version
+
+When a new version of an API is released, we should also update `nb-tester/requirements.txt` to ensure that our notebooks still work with the latest version of the API. You can do this upgrade either manually or wait for Dependabot's automated PR.
+
+Dependabot will fail to run at first due to not having access to the token. To fix this, have someone with write access push a commit to the PR, such as by merging main or using `git commit --allow-empty`.
```

Comment:
```suggestion
Dependabot will fail to run at first due to not having access to the token. To fix this, have someone with write access trigger CI for the PR, such as by merging main or closing then reopening the issue.
```
Closing/reopening is a bit easier since you can do it from GitHub.